src/openrct2-ui/WindowManager.cpp

```diff
@@ -156,10 +156,7 @@ class WindowManager final : public IWindowManager
                 {
                     return this->OpenWindow(WC_RESEARCH);
                 }
-                else
-                {
-                    return window_new_ride_open_research();
-                }
+                return window_new_ride_open_research();
```

Comment:
I prefer a space after an if block. This is, of course, a matter of preference, so feel free to decide yourself.

Reply:
It's pretty mixed here. I decided to add spaces when subsequence if-statements check for different things, while keeping it as is when they check the same values. In this case it's inside a switch-case where no other cases have if-statements or empty lines, so I went without adding one.